daemons/execd/execd_commands.c

```diff
@@ -1233,7 +1250,10 @@ lrmd_rsc_execute_stonith(lrmd_rsc_t * rsc, lrmd_cmd_t * cmd)
         }
     }
 
-    stonith_action_complete(cmd, rc);
+    stonith_action_complete(cmd,
+                            ((rc == pcmk_ok)? CRM_EX_OK : CRM_EX_ERROR),
+                            stonith__legacy2status(rc),
+                            rc == -pcmk_err_generic? NULL : pcmk_strerror(rc));
```

Comment:
I never remember the binding order with the ternary operator. But in any case it is inconsistent here.

Reply:
Probably to keep it under 80 characters ;) but consistency is probably more important